abc.templates/infra/spec.yaml

```diff
@@ -32,6 +32,9 @@ inputs:
     desc: 'The CI service account email for deploying GitHub Metrics Aggregator'
     rules:
       - rule: 'gcp_matches_service_account(automation_service_account_email)'
+  - name: 'terraform_actuator_members'
```

Comment:
dislike the name, but can't think of better

Reply:
Fine for now, do we need a list here tho? Ideally this is a single service account used by terraform. I'm not sure I have an opinion just posing the question.